src/analyses/extract_osek.ml

```diff
@@ -321,7 +321,7 @@ struct
   let threadspawn ctx lval f args fctx = ctx.local
   let exitstate  v = D.bot ()
 
-  let init () = (* registers which functions to extract and writes out their definitions *)
+  let init marshal = (* registers which functions to extract and writes out their definitions *)
```

Comment:
In principle, there is the same oddity as with `extract_arinc`: The `init ()` defined before is also not called here.